setup.py

```diff
@@ -12,7 +12,7 @@
 setuptools.setup(
     name='lib_logging',
     install_requires=[
-        "flake8~=3.8",
```

Comment:
Just a question. Do we still need this dependency? I will approve the PR nevertheless.

Reply:
Explained in https://github.com/xmos/lib_i2s/pull/140 :)